cartridges/app_storefront_pdp_extend/cartridge/scripts/helpers/notificationHelpers.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+function getClientContentAsset(contentAssetID) {
+    var ContentMgr = require('dw/content/ContentMgr');
+    var customerContent = ContentMgr.getContent(contentAssetID);
```

Comment:
This function seems generic for ContentAssets hence may be in contentAssetHelper